micronota/parsers/cmscan.py

```diff
@@ -0,0 +1,413 @@
+                       fields[_COLUMNS['SEQUENCE_START_POSITION']['position']])
+                      )
+
+            hitEnd = -1
+            if fields[_COLUMNS['SEQUENCE_END_POSITION']['position']].isdigit():
+                hitEnd = int(
+                   fields[_COLUMNS['SEQUENCE_END_POSITION']['position']]
+                )
+            else:
+                raise CmscanFormatError("%s %i %s '%s'." % (
+                        "Column",
+                        _COLUMNS['SEQUENCE_END_POSITION']['position'],
+                        "must be an integer value for the end position of the "
+                        "hit. Here, it is",
+                        fields[_COLUMNS['SEQUENCE_END_POSITION']['position']],
+                        ".")
+                      )
+
+            hitOrientation = fields[_COLUMNS['STRAND']['position']]
+            if hitOrientation == "+":
+                if hitStart > hitEnd:
+                    raise CmscanFormatError('%s %i %s %i %s' % (
+                          "On the forward strand (+), start position of a hit "
+                          "must always be smaller than its end position."
+                          " This is not true for the hit between",
+                          hitStart,
+                          "and",
+                          hitEnd,
+                          ". It might be, that this hit is in fact on the "
+                          "reverse strand. Please check strand orientation and"
+                          " positions."))
+            elif hitOrientation == "-":
+                if hitStart < hitEnd:
+                    raise CmscanFormatError('%s %i %s %i %s' % (
+                          "On the reverse strand (-), start position of a hit "
+                          "must always be larger than its end position."
+                          " This is not true for the hit between",
+                          hitStart,
+                          "and",
+                          hitEnd,
+                          ". It might be, that this hit is in fact on the "
+                          "forward strand. Please check strand orientation and"
+                          " positions."))
+                else:
+                    hitStart, hitEnd = hitEnd, hitStart  # swap orientation
+            else:
+                raise CmscanFormatError(
+                    "%s '%s' %s %i. %s" %
+                    ("Unknown strand character",
+                     hitOrientation, "in column",
+                     _COLUMNS['STRAND']['position'],
+                     "Valid characters are '+' for the forward strand and "
+                     "'-' for the reverse strand."))
+
+#           since Infernal want the user to be aware of differences between
+#           CMsearch and CMscan the information about model and query are at
+#           different columns. Here, we contradict this design and store the
+#           information always at the same key
```

Comment:
So indent text with the `if` statement below, and make sure that there is only 1 space between comment and `#`